### PR TITLE
feat(rviz_plugin): add target object type display

### PR DESCRIPTION
**Pull request by TakaHoribe**
## Description

Add a display to confirm which modules are using which types of the dynamic object. 
image

## Related links
None

## Tests performed

Run it with Psim.

## Notes for reviewers

None
## Interface changes

None

## Effects on system behavior

None

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].
- [x] The PR has been properly tested.
- [x] The PR has been reviewed by the code owners.

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [x] There are no open discussions or they are tracked via tickets.
- [x] The PR is ready for merge.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Comment by TakaHoribe** [comment]
@satoshi-ota Would you review this PR?

File: common/tier4_target_object_type_rviz_plugin/src/target_object_type_panel.cpp [hunk 1]
```diff
@@ -0,0 +1,248 @@
+//  Copyright 2023 Tier IV, Inc. All rights reserved.
```
Comment (satoshi-ota) [inline common/tier4_target_object_type_rviz_plugin/src/target_object_type_panel.cpp:1]:
```suggestion
//  Copyright 2023 TIER IV, Inc. All rights reserved.
```

File: common/tier4_target_object_type_rviz_plugin/src/target_object_type_panel.hpp [hunk 2]
```diff
@@ -0,0 +1,60 @@
+//  Copyright 2023 Tier IV, Inc. All rights reserved.
```
Comment (satoshi-ota) [inline common/tier4_target_object_type_rviz_plugin/src/target_object_type_panel.hpp:1]:
```suggestion
//  Copyright 2023 TIER IV, Inc. All rights reserved.
```

**Review by satoshi-ota — APPROVED** [review]
Cool!! In near future, I'd like to align namespaces.

**Pull request merged**